cheroot/__main__.py

```diff
@@ -65,7 +87,7 @@ def main(args=None):
         help='WSGI application callable to use')
     parser.add_argument(
         '--bind', metavar='ADDRESS',
-        dest='bind_addr', type=wsgi_bind_addr,
+        dest='_bind_addr', type=str,
```

Comment:
Why not just rely on automatically created property names? Why abuse underscore at all?

Reply:
1.) Explicit is better than implicit
2.) The use of underscore for the arguments was to make it possible to package up the majority of arguments from argparse using a simple loop rather than individually.

Basically if I unroll the loop, that will mean at least one if statement per argument to check for the None condition before adding it to the arguments list, which normally doesn't pass my definition of "beautiful is better than ugly". The other option would be to duplicate the class defaults in the argument parser, which hits up against the koan of "don't repeat yourself". Initially I thought the issue was with using the dictionary generator, but what I'm taking away now is that you'd prefer the args manually parsed to the Server() class. I'd like to hear your thoughts on which approach is the best course.